app/views/race_entries/_entries.html.erb

```diff
@@ -1,15 +1,15 @@
 <table class="table">
   <tr>
-    <th><%= link_to 'Racer', race_entries_path(sort: 'racers.last_name') %></th>
+    <th><%= link_to 'Racer', request.params.merge(sort: 'racers.last_name') %></th>
```

Comment:
Using `request.params.merge` allows us to add or change the `sort` param without affecting the rest of the URL. Because of this, we don't need to specify a URL path. This will just use the existing one and append (or change) the `sort` param on the existing path.